CoreNotesView/release/modules/ext/docs/CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/notes-view.md

```diff
@@ -87,3 +87,31 @@ view:
 -   *Time range filter*: A time range filter that allows the user to only  
     display notes in a specific time range. Times can be changed to different
     time zones.
+
+## Markdown Support
+
+The notes view supports some markdown text now. There is a checkbox in the new note
```

Comment:
capitalise "Notes View". I would potentially also reword from "some markdown text now" to "some Markdown formatting options"